source/index.js

```diff
@@ -57,6 +58,32 @@ module.exports = async (input = 'patch', options) => {
 	const hasLockFile = fs.existsSync(path.resolve(rootDir, 'package-lock.json')) || fs.existsSync(path.resolve(rootDir, 'npm-shrinkwrap.json'));
 	const isOnGitHub = options.repoUrl && hostedGitInfo.fromUrl(options.repoUrl).type === 'github';
 
+	let isPublished = false;
+
+	const rollback = onetime(async () => {
```

Comment:
Should we move `rollback()` to `source/git-util.js`? We could also create methods for the separate Git commands instead (i.e. `getLastCommit`, `deleteTag`, `removeLastCommit`).

Reply:
> We could also create separate methods for the Git commands instead (i.e. getLastCommit, deleteTag, removeLastCommit).

Was going to comment about that. Definitely yes.

> Should we move rollback() to source/git-util.js?

Not sure there will be enough left worth moving once the above is done. Let's do that first and we can see.